common/setups/rasr/hybrid_system.py

```diff
@@ -561,6 +480,29 @@ def nn_recog(
                     **r_args,
                 )
 
+    def nn_compile_graph(
+        self,
+        name: str,
+        returnn_config: returnn.ReturnnConfig,
+        epoch: Optional[int] = None,
+    ):
+        """
+        graph compile helper including alias
+
+        :param name:
+        :param returnn_config:
+        :param epoch:
+        :return: the TF graph
+        """
+        graph_compile_job = returnn.CompileTFGraphJob(
+            returnn_config,
+            epoch=epoch,
+            returnn_root=self.returnn_root,
+            returnn_python_exe=self.returnn_python_exe,
+        )
+        graph_compile_job.add_alias(f"nn_recog/graph/{name}.meta")
```

Comment:
the alias is for a folder.. so maybe not let the name end on ".meta" because this might suggest this is the compiled meta file.

Reply:
Ah this was already in the old code, but yes this is no excuse, will fix.